specificatie/openapi.yaml

```diff
@@ -558,6 +600,127 @@ components:
         pattern: '^[0-9]{16}$'
         example: '0226010000038820'
 
+    woonplaatsIdentificatie-query:
+      description: "De identificatie van een woonplaats."
+      name: woonplaatsIdentificatie
+      in: query
+      required: false
+      schema:
+        type: string
+        pattern: '^[0-9]{3}[1-9]|[0-9]{2}[1-9][0-9]|[0-9][1-9][0-9]{2}|[1-9][0-9]{3}$'
```

Comment:
is zo'n ingewikkeld pattern nodig/nuttig voor de clients?

Ik zou voorstellen:
pattern: '^[0-9]{4}$'

Reply:
Officieel kan een woonplaats ID niet 0000 zijn, vandaar deze regex. Dit stond er al zo in en laat ik voor nu zo.